src/base_classes/baseball.py

```diff
@@ -0,0 +1,165 @@
+"""
+Baseball Base Classes
+
+This module provides baseball-specific base classes that extend the core sports functionality
+with baseball-specific logic for innings, outs, bases, strikes, balls, etc.
+"""
+
+from typing import Dict, Any, Optional, List
+from src.base_classes.sports import SportsCore
+from src.base_classes.api_extractors import ESPNBaseballExtractor
+from src.base_classes.data_sources import ESPNDataSource, MLBAPIDataSource
+import logging
+
+class Baseball(SportsCore):
+    """Base class for baseball sports with common functionality."""
+    
+    # Baseball sport configuration (moved from sport_configs.py)
+    SPORT_CONFIG = {
+        'update_cadence': 'daily',
+        'season_length': 162,
+        'games_per_week': 6,
+        'api_endpoints': ['scoreboard', 'standings', 'stats'],
+        'sport_specific_fields': ['inning', 'outs', 'bases', 'strikes', 'balls', 'pitcher', 'batter'],
+        'update_interval_seconds': 30,
+        'logo_dir': 'assets/sports/mlb_logos',
+        'show_records': True,
+        'show_ranking': True,
+        'show_odds': True,
+        'data_source_type': 'espn',  # Can be overridden for MLB API
+        'api_base_url': 'https://site.api.espn.com/apis/site/v2/sports/baseball'
```

Comment:
U have this value but then u also have https://github.com/ChuckBuilds/LEDMatrix/pull/63/files#diff-cdbedce753047a0d0dbbf06505964478da83060dfaf230c5bfd21275f35c974cR63 which seems to contradict things or duplicative. AND also https://github.com/ChuckBuilds/LEDMatrix/pull/63/files#diff-b15dd7f9659df01eb02511a63f63d3a6ff5ae45a17cab0906a4327bb829de6e4R15 which is being used in the background fetch functions which don't seem to be using the new logic u made?